src/ampycloud/data.py

```diff
@@ -361,12 +380,11 @@ def _setup_sligrolay_pdf(self, which: str = 'slices') -> tuple[pd.DataFrame, npt
                 ]
 
         # We want to raise early if 'which' is unknown.
-        if not which in ['slices', 'groups', 'layers']:
+        if which not in ['slices', 'groups', 'layers']:
```

Comment:
Was this a bug or does it have the same behaviour?

Reply:
Not a bug and the behavior was identical, but the new version is better Python syntax.

Reply:
yes, sorry, I first thought the pipeline failed because of too many pylint issues, so I went for some easy ones. But then I realized it was due to `setuptools` missing as a dependency with Python 3.12. Nevertheless, I'd keep these little style adjustment to comply better with PEP standards.